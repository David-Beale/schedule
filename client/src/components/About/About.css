.about-container {
  margin-top: 100px;
  text-align: center;
  width: 60%;
  margin: auto;
}

.about-container h2 {
  color: orange;
  margin: 24px auto;
}
.team__container {
  display: flex;
  width: 100%;
  justify-content: space-evenly;
}
.dev__container {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}
.dev__container h5 {
  padding: 6px;
}

.about-section {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: 30px;
}
