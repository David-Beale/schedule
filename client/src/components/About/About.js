import React from 'react';
import db from '../../assets/logos/db.jpg';
import pp from '../../assets/logos/pp.jpg';
import gs from '../../assets/logos/gs.jpg';
import jt from '../../assets/logos/jt.jpg';

export default function About() {
  return (
    <div className='about-container'>
      <div className='about-section'>
        <h2>Who are we</h2>
        <p>
          software engineers that share the same belief of helping passionate
          people to share their skills and passions.
        </p>
        <p>
          We believe that the current situation shouldn't prevent anyone to
          share skills and knowlegde
        </p>
        <p>
          We want to help everyone to be able to start and continue sharing
          their skills and abilities and knowledge through the means they
          already have or help them create new ways to share.
        </p>
      </div>
      <div className='about-section'>
        <h2>How we do it</h2>
        <p>
          We are sharing anyone's events so that everyone knows when their
          favorite performer is going to share again. We keep track of the time
          of events so that no one would miss events that they would like to
          attent to.
        </p>
      </div>
      <div className='about-section'>
        <h2>What we do</h2>
        <p>
          We gather information from people's submission through our platform
          with the details of their events and display them to the public.
        </p>
      </div>
      <div className='about-section'>
        <h2>The Team</h2>
        <div className='team__container'>
          <div className='dev__container'>
            <img src={db} alt='David Beale' />
            <h5>David Beale</h5>
          </div>
          <div className='dev__container'>
            <img src={pp} alt='Pawel Pietruszka' />
            <h5>Pawel Pietruszka</h5>
          </div>
          <div className='dev__container'>
            <img src={gs} alt='Giovanni Stasi' />
            <h5>Giovanni Stasi</h5>
          </div>
          <div className='dev__container'>
            <img src={jt} alt='Joseph Tolentino' />
            <h5>Joseph Tolentino</h5>
          </div>
        </div>
      </div>
    </div>
  );
}
