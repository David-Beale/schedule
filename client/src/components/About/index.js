import About from './About';
import './About.css';

export { About as default };
