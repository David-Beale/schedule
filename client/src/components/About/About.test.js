import About from './index';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('About', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
  });
  it('should render properly', () => {
    const about = render(<About />);
    expect(about).toMatchSnapshot();
  });
});
