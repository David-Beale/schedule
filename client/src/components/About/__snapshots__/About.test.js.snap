// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="about-container"
      >
        <div
          class="about-section"
        >
          <h2>
            Who are we
          </h2>
          <p>
            software engineers that share the same belief of helping passionate people to share their skills and passions.
          </p>
          <p>
            We believe that the current situation shouldn't prevent anyone to share skills and knowlegde
          </p>
          <p>
            We want to help everyone to be able to start and continue sharing their skills and abilities and knowledge through the means they already have or help them create new ways to share.
          </p>
        </div>
        <div
          class="about-section"
        >
          <h2>
            How we do it
          </h2>
          <p>
            We are sharing anyone's events so that everyone knows when their favorite performer is going to share again. We keep track of the time of events so that no one would miss events that they would like to attent to.
          </p>
        </div>
        <div
          class="about-section"
        >
          <h2>
            What we do
          </h2>
          <p>
            We gather information from people's submission through our platform with the details of their events and display them to the public.
          </p>
        </div>
        <div
          class="about-section"
        >
          <h2>
            The Team
          </h2>
          <div
            class="team__container"
          >
            <div
              class="dev__container"
            >
              <img
                alt="David Beale"
                src="db.jpg"
              />
              <h5>
                David Beale
              </h5>
            </div>
            <div
              class="dev__container"
            >
              <img
                alt="Pawel Pietruszka"
                src="pp.jpg"
              />
              <h5>
                Pawel Pietruszka
              </h5>
            </div>
            <div
              class="dev__container"
            >
              <img
                alt="Giovanni Stasi"
                src="gs.jpg"
              />
              <h5>
                Giovanni Stasi
              </h5>
            </div>
            <div
              class="dev__container"
            >
              <img
                alt="Joseph Tolentino"
                src="jt.jpg"
              />
              <h5>
                Joseph Tolentino
              </h5>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="about-container"
    >
      <div
        class="about-section"
      >
        <h2>
          Who are we
        </h2>
        <p>
          software engineers that share the same belief of helping passionate people to share their skills and passions.
        </p>
        <p>
          We believe that the current situation shouldn't prevent anyone to share skills and knowlegde
        </p>
        <p>
          We want to help everyone to be able to start and continue sharing their skills and abilities and knowledge through the means they already have or help them create new ways to share.
        </p>
      </div>
      <div
        class="about-section"
      >
        <h2>
          How we do it
        </h2>
        <p>
          We are sharing anyone's events so that everyone knows when their favorite performer is going to share again. We keep track of the time of events so that no one would miss events that they would like to attent to.
        </p>
      </div>
      <div
        class="about-section"
      >
        <h2>
          What we do
        </h2>
        <p>
          We gather information from people's submission through our platform with the details of their events and display them to the public.
        </p>
      </div>
      <div
        class="about-section"
      >
        <h2>
          The Team
        </h2>
        <div
          class="team__container"
        >
          <div
            class="dev__container"
          >
            <img
              alt="David Beale"
              src="db.jpg"
            />
            <h5>
              David Beale
            </h5>
          </div>
          <div
            class="dev__container"
          >
            <img
              alt="Pawel Pietruszka"
              src="pp.jpg"
            />
            <h5>
              Pawel Pietruszka
            </h5>
          </div>
          <div
            class="dev__container"
          >
            <img
              alt="Giovanni Stasi"
              src="gs.jpg"
            />
            <h5>
              Giovanni Stasi
            </h5>
          </div>
          <div
            class="dev__container"
          >
            <img
              alt="Joseph Tolentino"
              src="jt.jpg"
            />
            <h5>
              Joseph Tolentino
            </h5>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
